tools/bazel_integration_test/test_runner.py

```diff
@@ -52,6 +52,19 @@ def main(conf_file):
                     "--override_repository=rules_python=%s/rules_python"
                     % os.environ["TEST_SRCDIR"]
                 )
+                bazel_args.append(
+                    "--override_module=rules_python=%s/rules_python"
+                    % os.environ["TEST_SRCDIR"]
+                )
+
+                # TODO: Despite not declaring dependencies on these in MODUILE.bzl
+                # or in the bzlmod example's MODULE.bzl, bazel is complaining about
+                # the following versions being yanked. For now, allow all yanked
+                # versions in integration tests.
+                #
+                #   protobuf@3.19.2
+                #   zlib@1.2.12
+                bazel_args.append("--allow_yanked_versions=all")
```

Comment:
@Wyverald Any idea why this was required? Those versions are indeed yanked in the BCR, but I don't have any direct or transitive dependencies on them via bzlmod (that I know of).

Reply:
It works with `last_green` and was fixed by https://github.com/bazelbuild/bazel/commit/d6d6183dc17484a8d5addcd68a7480ce7a5a97f6. While your module didn't depend on the yanked `zlib`, `@bazel_tools` did.

Reply:
Yep -- `bazel_tools` is a built-in module that depends on stuff like `rules_cc`, `rules_java`, etc. Unfortunately it also depends on these two yanked versions, but using 6.0.0rc1 should fix this.

Reply:
That works, thanks!